workflows/rnaseq/rnaseq-core.wdl

```diff
@@ -55,18 +55,32 @@ workflow rnaseq_core {
     }
 
     input {
-        Array[File] read_one_fastqs_gz
-        Array[File] read_two_fastqs_gz
         File gtf
         File star_db
+        Array[File] read_one_fastqs_gz
+        Array[File] read_two_fastqs_gz
         String read_groups
         String prefix
         File? contaminant_db
+        SJ_Motifs alignSJstitchMismatchNmax = SJ_Motifs {
```

Comment:
2 small changes I'd like to see happen here (in `rnaseq-core`)

1. I can be persuaded that this is overkill, but I would prefix all these new params with `star_`. I'd just like it to be clear that these parameters are specific to the STAR alignment task.
2. we're missing parameter meta for all of these. I think a copy+paste from STAR is fine, with two small tweaks.
    - The first sentence needs to provide context by stating that the param impacts STAR alignment
    - please add a `tool_default:` key that makes it clear _this default_ is overriding the _tool/task/STAR default_. IDK if "tool"_default is correct term there.

Reply:
Based on Slack conversation, we decided to skip the `star_` prefix, but add a `tool: star` meta entry to make it clear what the underlying tool is.